services/src/main/java/org/keycloak/connections/httpclient/HttpClientBuilder.java

```diff
@@ -101,9 +101,9 @@ public void checkServerTrusted(X509Certificate[] chain,
     protected long maxConnectionIdleTime = 900000;
     protected TimeUnit maxConnectionIdleTimeUnit = TimeUnit.MILLISECONDS;
     protected HostnameVerifier verifier = null;
-    protected long socketTimeout = -1;
+    protected int socketTimeout = -1;
```

Comment:
Why this change?

Reply:
Same as [this](https://github.com/keycloak/keycloak/pull/8165#discussion_r658516576).